### ISSUE TITLE
edc_capitalprojects concat with previous version

### ISSUE THREAD

**Issue by td928** [NYCPlanning/db-data-library#216]
currently no established process for concat the email updated edc_capitalprojects with the previous version. 

Enhancement: 
use the scriptor process to make this integrated with established process like other data library updates.

**Comment by td928** [comment]
#216 has most up to date working on this

the next steps would be:

1. finished the `edc_capitalprojects.py` script that would pull the loaded previous latest version table from recipe engine. 
2. join with the new table ingested with geopandas under local `library\tmp\` dir
3. write the joined table to `library\tmp\EDCCapital`
4. update the `edc_capitalprojects.yml` template to run the python script before the uploading step. Also add the note in the template that `edc_capitalprojects.sh` should be run before step 1
5. use data library as normal to run the workflow

**Comment by mbh329** [comment]
Hey @td928 I remember I was the one who uploaded this dataset to data libraries, if you have a quick second today, can you walk me through this issue?

**Comment by td928** [comment]
@mbh329 yeah for sure! although I might wait until we make a decision on whether this work will stay. Amanda is going to look at my work so far and determine whether it would be worth the efforts to do all the steps listed above.

**Comment by mbh329** [comment]
Sounds good
